pages/posts/w3c-wot-case-study.md

```diff
@@ -0,0 +1,141 @@
+talk to each other due to differences in data interchange mechanisms.
+
+To integrate these disparate devices, developers must work with a growing set
+of protocols, serialization formats and API specifications. This results in
+repetitive, non-scalable and error-prone work that is difficult to automate.
+
+While technologies like OpenAPI and AsyncAPI largely solve this problem in the
+context of Web APIs, they fall short for describing networks of non-HTTP
+multi-protocol devices and do not consider different modes of interaction based
+on their meaning in the physical world.
+
+## Solution
+
+To solve these problems, the W3C Web of Things works on providing standardized
+building blocks that make use of JSON Schema.
+
+JSON Schema is used for validating descriptions of network-facing capabilities
+of physical entities called [Thing Descriptions][thing-description], and to
+model and describe data sent by Internet of Things consumers and producers in a
+multi-protocol manner.
+
+The W3C Web of Things specifications are on board with JSON Schema since Draft
+4, and even the first draft versions already catered to data models of Internet
+of Things devices.  "Currently, we are on JSON Schema Draft 7 and expect to
+move to JSON Schema
+[2020-12](https://json-schema.org/specification-links.html#understanding-draft-names-and-numbers)
+or a newer one when starting our new charter in February 2023," said Ege
+Korkan, Research Scientist at Siemens and W3C Specifications editor.
+
+"We observe JSON Schema becoming more and more stable over the years and
+Postman's support to the community gives us even more confidence on it,"
+continued Ege Korkan.
+
+JSON Schema is delivering exciting features, and more importantly these
+features are becoming more and more consistent across implementations.  For
+example, Ege Korkan added, "vocabularies are very promising and we plan to
+explore them further in conjunction with Semantic Web technologies."
+
+## Impact
+
+The W3C Web of Things relies on JSON-LD for linking vocabularies and ontologies
+that already exist on the web, such as [SAREF][saref], [Units of
+Measure][units-of-measure] and [Schema.org][schema-org]. Being based on the
+JSON data model, JSON Schema fits well with JSON-LD when integrating both
+technologies as part of a single specification.
+
+Adopting JSON Schema means that the W3C Web of Things does not need to invest
+effort on inventing another schema language. As a consequence of the popularity
+of JSON Schema, existing tooling can be often reused instead of implementing
+custom parsers, validators, code generators and UI generators for all the
+programming languages that the community might be interested in.
+
+JSON Schema has proven to be a foundational block for creating higher-level
+tooling that is specific to the W3C Web of Things. As a notable example, the
+community has developed a tool that makes use of JSON Schema definitions inside
```

Comment:
Is it worth linking to the tool?

Reply:
I don't have the link. @egekorkan is this public?

Reply:
Here it is https://github.com/tum-esi/testbench